upgrade/UPGRADE-v13.0.0-dev.md

```diff
@@ -46,3 +46,7 @@ There you can find links to upgrade notes for other versions too.
         - docker/php-fpm/docker-php-entrypoint
         - docker/php-fpm/phing-completion
     - see #project-base-diff to update your project
+- remove usage of article placement ([#2776](https://github.com/shopsys/shopsys/pull/2776))
+    - placement `Shopsys\FrameworkBundle\Model\Article\Article::PLACEMENT_TOP_MENU` was removed along with the usages
+    - already existing articles with this placement will be migrated to `Shopsys\FrameworkBundle\Model\Article\Article::PLACEMENT_NONE` placement
+    - see #project-base-diff to update your project
```

Comment:
We may add notice like "If you use this article's placement, skip the migration XYZ"

Reply:
added